main.cpp

```diff
@@ -1 +1,32 @@
+#include <iostream> 
+#include <cstdlib>
+#include <iomanip>
+#include "coordinates.h"
+#include <vector> 
 
+using namespace std;
+
+int main() {
+    // loop that reads comma-delimited lines from standard input one line at a time and constructs a vector of coordinates 
+string coords;
+CoordVec x; 
+Coordinate userInput;
+// Using unix pipe to get coords for input  
+while (getline(cin, coords)){
+        // get whole line
+        int commaLocation = coords.find(",");
+        string lat = coords.substr(0, commaLocation -1); 
+        string longitude = coords.substr(commaLocation + 1, coords.back());
+        // push variable into a vector 
+        userInput.first = atof(lat.c_str());
```

Comment:
it didn't cross to my mind to use atof. Also, the way you used c_str to split characters of coordinate. Smart move!